docs/use/database_view.md

```diff
@@ -52,6 +52,12 @@ The Components page offers comprehensive information about pods and containers.
 
    ![!image](../images/everest_components_page.png)
 
+   The Components tab also includes an interactive Topology View that improves the visibility and management of DB clusters deployed in Percona Everest. This view visually represents the cluster's components, such as pods and services, along with their status and relationships.
+
+   Click the Toggle **Table View** to switch between the **Topology** and **Table** views.
+
+   ![!image](../images/Create components_page_diagram_view.png)
```

Comment:
the image seems to be lost, I can't find it in the PR and it's not rendered in the deployed version
<img width="762" alt="Screenshot 2025-05-23 at 13 17 01" src="https://github.com/user-attachments/assets/799441c9-63db-49a9-8ea0-11a30b189789" />